packages/mermaid-zenuml/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@mermaid-js/mermaid-zenuml",
+  "version": "0.1.0",
+  "description": "MermaidJS plugin for ZenUML integration",
+  "module": "dist/mermaid-zenuml.core.mjs",
+  "types": "dist/detector.d.ts",
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./dist/mermaid-zenuml.core.mjs",
+      "types": "./dist/detector.d.ts"
+    },
+    "./*": "./*"
+  },
+  "keywords": [
+    "diagram",
+    "markdown",
+    "zenuml",
+    "mermaid"
+  ],
+  "scripts": {
+    "prepublishOnly": "pnpm -w run build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mermaid-js/mermaid",
+    "directory": "packages/mermaid-zenuml"
+  },
+  "contributors": [
+    "Peng Xiao",
+    "Sidharth Vinod <github@sidharth.dev> (https://sidharth.dev)",
+    "Dong Cai"
```

Comment:
@sidharthv96  Oh, I don't want to make this email address `dongc@blackmagicdesign.com` public. Thanks for your guide. I am able to set up the private email `dontry@users.noreply.github.com` on settings page. I am wondering what would happen to those pushed commits associated with email address `dongc@blackmagicdesign.com`. Will Github automatically mask them with the private ones? Is it possible to alter the git history to remove it?

Reply:
> Will Github automatically mask them with my new private email?

No it won't, and even if it does, somebody could just `git clone` and see those details anyway. Git is a bit like a blockchain, so changing the old emails would change all the commit hashes.

There are some options for GitHub to reject your old email when doing a `git push` though, see https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/blocking-command-line-pushes-that-expose-your-personal-email-address

> Is it possible to alter the git history to remove it?

Yep! Once it's in `develop`/`master`, it won't be possible, but since this is just a small branch, I'd be happy to change it!

I've done a `git rebase` on this `sidv/zenuml` branch and changed the author of the commits using `git commit --amend --author="Dong Cai <3481791+dontry@users.noreply.github.com>" --no-edit`, so that once this PR is merged, your old email won't appear in the git history.